test/unit/v1beta1/tune-api/test_tune_api.py

```diff
@@ -0,0 +1,890 @@
+import unittest
```

Comment:
@helenxie-bit Let's add the unit tests under [the `katib_client_test.py` file](https://github.com/kubeflow/katib/blob/e4f7922ac448257485dcd6377d2b4aa663ba0d15/sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py#L295) since we previously agreed to place unit test files near the actual files.
You can use the tests data for `tune` API and define the fixture, similar to other SDK APIs.

Reply:
@andreyvelich Thank you! I have followed this instruction and updated unit tests for `tune` API. Please review it when you have time.